### PR TITLE
Add search input to the header

### PR DESCRIPTION
**Pull request by gl2748**
Supersedes:  https://github.com/steemit/condenser/pull/1992
I have reviewed and tested locally - all is good.

### CODE REVIEW

**Comment by bnchdrff** [comment]
the search box has a border when focused in firefox:

screenshot from 2017-12-09 12-15-47

File: src/app/components/modules/TopRightMenu.jsx [hunk 1]
```diff
@@ -122,7 +122,7 @@ function TopRightMenu({
                         )}
                     </a>
                 </li>
-                {!pathCheck ? submit_story : null}
```
Comment (bnchdrff) [inline src/app/components/modules/TopRightMenu.jsx]: why is this changed?

Comment (gl2748) [reply src/app/components/modules/TopRightMenu.jsx]: good spot, not so sure, undoing...

**Comment by gl2748** [comment]
@bnchdrff addressed your comment.
